exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,10 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+var percentageOfStudent = Math.round((numberOfStudents/total) * 100);
+console.log( "percentage of student: " + percentageOfStudent );
+var percentageOfMentors = Math.round((numberOfMentors/total) * 100);
+console.log("percentage of mentors: " + percentageOfMentors);
```

Comment:
I would have just added a % sign at the end of each log message. Other than that very good!